core/src/main/java/org/web3j/service/TxHSMSignService.java

```diff
@@ -56,7 +56,6 @@ public byte[] sign(RawTransaction rawTransaction, long chainId) {
         if (isLegacy) {
             signatureData = createEip155SignatureData(signatureData, chainId);
         }
-
```

Comment:
Can we remove this change, it looks a bit confusing of a change in TxHSMSignService.java file